tools/plot/episode_loader.py

```diff
@@ -17,6 +17,10 @@
 import numpy as np
 import torch
 
+
+class TimestepMappingError(Exception):
```

Comment:
This should inherit from ValueError, since it's a special case of that. This allows a direct or indirect caller further up the call stack to catch all ValueErrors and still catch this one, or be more specific and catch just this one. When it inherits from Exception, we lose that sort flexibility.
```suggestion
class TimestepMappingError(ValueError):
```